util/subprocesses.py

```diff
@@ -79,8 +79,10 @@ def getFreeSpace(folder, mulVar):
 
 
 def captureStdout(inputCmd, ignoreStderr=False, combineStderr=False, ignoreExitCode=False,
-                  currWorkingDir=os.getcwdu(), env='NOTSET', verbosity=False):
+                  currWorkingDir=None, env='NOTSET', verbosity=False):
     """Capture standard output, return the output as a string, along with the return value."""
+    currWorkingDir = currWorkingDir or (
+        os.getcwd().decode("utf-8", errors="replace") if sys.version_info.major == 2 else os.getcwd())
```

Comment:
Ditto re: `getcwd().decode()` vs `getcwdu()`

Nice use of `None` and `or` though 👍